src/p2p/net_node.h

```diff
@@ -118,6 +118,8 @@ namespace nodetool
         m_in_timedsync(false)
     {}
 
+    static constexpr int handshake_command() noexcept { return 1001; }
```

Comment:
Can that be COMMAND_HANDSHAKE::ID ?

Reply:
`COMMAND_HANDSHAKE` is a `typedef` for `COMMAND_HANDSHAKE_T<t_payload_net_handler::payload_type>` inside of `node_server`, and the type `t_payload_net_handler` is a template argument to `node_server`. This is outside of the `node_server` class entirely, so I decided to just hard-code and add a `static_assert`.

Instead, I will do `{ return COMMAND_HANDSHAKE_T<int>::ID; }` with a comment, and the `static_assert` to ensure no specializations.

Reply:
Or wait - is it just easier to leave as `{ return 1001; }` with the `static_assert`? Its unfortunate, but catches mistakes at build time.

Reply:
Fine by me to leave as is if's not that simple as just replacing.